docs/ENV.md

```diff
@@ -0,0 +1,58 @@
+# Environment variables
+
+##### Required
+
+Name | Example | Description |
+---- | ------- | ----------- |
+`API_HOSTNAME` | `wallet.com` | API public hostname.
+`API_PORT` | `3000` | API private port (used as both public and private port if `API_PUBLIC_PORT` is not specified).
+`API_DB_URI` | `postgres://localhost/wallet` | URI for connecting to a database.
+`API_LEDGER_ADMIN_USER` | `admin` | Ledger admin username.
+`API_LEDGER_ADMIN_PASS` | `pass` | Ledger admin pass.
+`CLIENT_HOST` | `wallet.com` | Publicly visible hostname.
```

Comment:
There shouldn't be any `_HOST`, `_HOSTNAME`, `_PUBLIC_HTTPS`, `_PUBLIC_PORT` or `_PUBLIC_PATH` fields. Everything should use `*_PUBLIC_URI` like the ledger.